qemu/tests/driver_in_use.py

```diff
@@ -20,7 +20,8 @@ def check_bg_running(vm, params):
     session = vm.wait_for_login()
     target_process = params.get("target_process", "")
     if params['os_type'] == 'linux':
-        output = session.cmd_output_safe('pgrep -l %s' % target_process)
+        output = session.cmd_output_safe('pgrep -l %s' %
+                                         target_process, timeout=60)
```

Comment:
`timeout=60` is the default setting for `cmd_output_safe`.

Reply:
will delete this.